appdatabase/migrations/sql/0012_pending_transactions.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS pending_transactions (
+  network_id UNSIGNED BIGINT NOT NULL,
+  transaction_hash VARCHAR NOT NULL,
+  blk_number BIGINT NOT NULL,
+  address VARCHAR NOT NULL,
```

Comment:
Could you add a `to_address` and a `from_address` to allow for both in bound and out bound pending transactions to be stored?